tensorboard/plugins/mesh/mesh_demo.py

```diff
@@ -0,0 +1,99 @@
+"""Simple demo which displays constant 3D mesh."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import numpy as np
+import tensorflow as tf
+import trimesh
```

Comment:
this dependency does not exist on our project I believe. Perhaps catch the failure and prompt user to install trimesh to run this? @nfelt would have better idea than mine :)

Reply:
can we add it to tensorboard or you'd rather not?
Also do you think I need build rule for demo app?